File: 240306/두 번째를 첫 번째로/second-to-first.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    string x;
    cin >> x;
    char first = x[0], second = x[1];
    for (int i=1; i<x.length(); i++){
        if (x[i] == second){
            x[i] = first;
        }
    }
    cout << x;
    return 0;
}

File: 240611/두 번째를 첫 번째로/second-to-first.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String str = sc.next();
        char first = str.charAt(0);
        char second = str.charAt(1);

        char[] ss = str.toCharArray();

        for (int i = 0; i<ss.length; i++){
            if (ss[i] == second)
                ss[i] = first;
        }

        System.out.print(String.valueOf(ss));

    }
}